optimum/exporters/tasks.py

```diff
@@ -735,39 +747,54 @@ def infer_task_from_model(model_name_or_path: str) -> str:
 
     @staticmethod
     def get_model_from_task(
-        task: str, model: str, framework: str = None, cache_dir: str = None
+        task: str,
+        model_name: str,
+        subfolder: str = "",
+        revision: Optional[str] = None,
+        framework: Optional[str] = None,
+        cache_dir: Optional[str] = None,
+        **model_kwargs
     ) -> Union["PreTrainedModel", "TFPreTrainedModel"]:
         """
         Retrieves a model from its name and the task to be enabled.
 
         Args:
             task (`str`):
                 The task required.
-            model (`str`):
+            model_name (`str`):
                 The name of the model to export.
-            framework (`str`, *optional*):
+            subfolder (`str`, *optional*, defaults to `""`):
+                The subfolder where the model is stored.
```

Comment:
```suggestion
                In case the relevant files are located inside a subfolder of the model repo on huggingface.co, you can
                specify the folder name here.
```